android/src/test/java/com/segment/analytics/kotlin/android/AndroidLifecyclePluginTests.kt

```diff
@@ -364,7 +345,7 @@ class AndroidLifecyclePluginTests {
         // Simulate activity startup
         lifecyclePlugin.onActivityCreated(mockActivity, mockBundle)
 
-        assertTrue(mockPlugin.ran)
+        verify (timeout = 2000){  mockPlugin.updateState(true) }
```

Comment:
is this timeout necessary due to us not being able to run the tests with the `testDispatcher`?

Reply:
correct. because we are not able to use `testDispatcher`, things running async. there is a delay before the update happen. the `verify` waits until the statement happened or timeout reached, whichever one comes first.